dev-tools/mage/fips-settings.yaml

```diff
@@ -2,10 +2,15 @@
 beats:
   - agentbeat
   - auditbeat
-  - cloudbeat
```

Comment:
Why was this removed?

Reply:
I misread the list of the initial FIPS beat and thought this was not included. Will re-add it

Reply:
Makes sense; I had not checked that. Thanks.